prettyPrinters/rust_types.py

```diff
@@ -35,7 +35,7 @@ class RustType(object):
 
 # Should be synchronized with `RsDebugProcessConfigurationHelper.RUST_STD_TYPES`
 STD_STRING_REGEX = re.compile(r"^(alloc::([a-z_]+::)+)String$")
-STD_STR_REGEX = re.compile(r"^&str$")
+STD_STR_REGEX = re.compile(r"^&?str\*?$")
```

Comment:
Actually, I cannot get `str*` type for MSVC LLDB
For the following code
```rust
fn main() {
    let mut s = String::from("Hello!");
    let x = s.as_mut();
    let y = x as *mut str;

    let a = "123";
    let b = a as *const str;
    println!("")
}
```
I got
![image](https://user-images.githubusercontent.com/2539310/115838917-d2f44800-a422-11eb-8341-e7ca9bdf3f7c.png)

i.e. `&str` without Rust support is rendered as `str` (at least for 1.51.0).
So you change works for msvc lldb but only because of `?` after `*`.

Reply:
That's interesting. Previously, I saw `str*` on Windows, but now I see `str` too.
I don't know why, but anyway, I generalized the implementation to handle all reference/pointer types